### ISSUE TITLE
Option leave off start patches

### ISSUE THREAD

**Issue by roman4e** [armin3600/orb-ores#1]
Hi. This is amazing mode I've been looking for years. It really removes starting ore patches. Every time it was generated too close to the center.

1. Is it possible to add options just to leave ore sizes and shapes as is, but remove starting ores patches?

2. Make an option to scatter around all starting patching on some random distance from N to M?

**Comment by armin3600** [comment]
I noticed this a few weeks ago when I tried playing it without space exploration. I originally made it to counteract the annoying ore patches SE makes that are wacky shapes so I tested it with that (and k2 due to the imersite caves). 

I tried fixing this but the 2 ideas I had for fixing it didn't work at all. I don't really have the patience to figure out how to fix them right now, I'm sure when I get back to playing factorio it'll annoy me enough that I will but in the meantime you're free to fix it and submit a PR, then I'll build and publish it to the portal.

as for 2, similar to 1 I don't know how to control starter patches. It's a good idea and I'll keep it in mind if I ever do go and figure it out but feel free to implement it yourself and submit a PR.

**Comment by roman4e** [comment]
I'll also continue investigating for it. Should be something about 'autoplace' settings, I've seen in other modules. It is about resource generation for sure, but you know there is almost no docs about how Factorio implemented it. Call me here if you find something.

**Comment by armin3600** [comment]
Yeah its entirely to do with autoplace settings, thats how the game decides how to place ores and stuff. You can see in this mod's files that I copied this https://github.com/wube/factorio-data/blob/master/core/lualib/resource-autoplace.lua and changed it a little to turn blobbiness to 0 (or false, I can't remember exactly it was a while ago). 

This may also be useful https://github.com/wube/factorio-data/blob/master/base/prototypes/entity/resources.lua
